lua/swenv/api.lua

```diff
@@ -117,6 +117,28 @@ M.get_venvs = function(venvs_path)
     })
   end
 
+  -- PYEVN
+  local pyenv_root = vim.fn.getenv('PYENV_ROOT')
+  if pyenv_root ~= vim.NIL then
+    local pyenv_versions_dir = Path:new(pyenv_root) .. "/versions"
+    local pyenv_versions_paths = scan_dir(pyenv_versions_dir, { depth = 1, only_dirs = true, silent = true })
```

Comment:
Thanks for the PR and sorry for the late reply! I've been very busy with other things.

This looks good to me but since this structure is now the same for all the frameworks I wonder if it would be good to separate the `scan_dir` and looping over paths to add to `venv` into separate functions which can be reused. In principle unrelated to this feature but let me know if that's something you would be willing to do.

Reply:
@AckslD It would be amazing to have pyenv support. Do you think @AckslD it's too much time for you to address the comment you left to @NeedleCats? I would be willing to take over but I am 0-experienced with lua in Neovim, I'd be happy to take over if I have a bit more instructions on the sentence:

> separate the scan_dir and looping over paths to add to venv into separate functions which can be reused

Thank you!